tests/netebpfext_unit/netebpf_ext_helper.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "netebpf_ext_helper.h"
+
+ebpf_registry_key_t ebpf_root_registry_key = HKEY_CURRENT_USER;
+DEVICE_OBJECT* _net_ebpf_ext_driver_device_object;
+
+_netebpf_ext_helper::_netebpf_ext_helper()
```

Comment:
Can this be refactored into a common code lib?

Reply:
Which part of it?

Reply:
Propose we refactor it into common code should the need arise.